translations/test/missing-fr-translations.json

```
@@ -0,0 +1,59 @@
+{
+  "catalog-import-test": {
```

```suggestion
  "catalog-import-test": {
```

This isn't a valid plugin-id, is it added for test?

---

Yes, this string was being translated, but there was no corresponding JSON object for it. So, instead of modifying the original files, I had to create a new plugin ID to include the translated string. I also needed a unique name, so I added the suffix `-test`.